### ISSUE TITLE
Outputting junk on error

### ISSUE THREAD

**Issue by JoshuaGreen** [thomas-maeder/popeye#298]
At least on my system, some bad input yields junk.&nbsp; Specifically, with version 4.85 on Windows,
```
BeginProblem
stip diag46
```
yields:
```
input-error:wrong integer
offending item: diag46
input-error:cannot recognize the stipulation
offending item: diag46
input-error:offending item: € T
offending item: diag46
```
While it's possible that this error output can be improved, my attention was drawn to the fifth line where it's clear that we're outputting junk.&nbsp; I'm not exactly sure what's going on here, especially as I don't see this when running on my Linux VM.&nbsp; (My guess is that it has something to do with the different line ending encodings on the various systems.)&nbsp; Regardless, we should fix this.

**Comment by JoshuaGreen** [comment]
Those unusual bytes are (on my Windows 7 system, running `pywin64.exe`): `0x80 0xA0 0x54 0x01`.&nbsp; That `0x80` looks _awfully_ suspicious.

**Comment by JoshuaGreen** [comment]
This has been (presumably) fixed in [`/commit/53e43de0be9b3126d74a7a7cb3d74fc20b606c2b`](https://github.com/thomas-maeder/popeye/commit/53e43de0be9b3126d74a7a7cb3d74fc20b606c2b).

**Comment by JoshuaGreen** [comment]
(It might be worth searching the codebase to see if there are any more cases where a `va_list` is being passed into a variadic function.)
